README.md

```diff
@@ -21,3 +21,76 @@ Optional flags:
 --reset-chain       Delete local data to do a clean connect to the testnet
 
 ```
+See https://hub.docker.com/repository/docker/axelarnet/axelar-core and https://hub.docker.com/repository/docker/axelarnet/tofnd for the latest available versions of the docker images.
+
+## Instructions to mint and burn tokens
+These instructions are a step by step guide to run manual commands for the minting and burning process.
+
+### Mint ERC20 wrapped Bitcoin tokens on Ethereum
+
+- Create a deposit address on Bitcoin
+  ```
+  axelarcli tx bitcoin link ethereum {ethereum addr, e.g. 0xc1c0c8D2131cC866834C6382096EaDFEf1af2F52} --from validator -y -b block
+  -> returns deposit address
+  ```
+- External: send BTC to deposit address and wait for 6 confirmations (i.e. the transaction is 6 blocks deep in the Bitcoin chain).
+- Create verification json object for Axelar
+  ```
+  axelarcli q bitcoin txInfo {blockhash, e.g. 4ac9dc50dc1b952cb1efca1e634216da2f5e3a12b4a4a802ce0f6b1271876bd2} {txID:voutIdx, e.g. da5b2e8037ce4b95f40ada01c6c2cd3ccb806d0a952906130eb9b806f7887590:1}
+  -> returns json of verification info for the given outpoint (copy the escaped string)
+  ```
+- Verify the Bitcoin outpoint
+  ```
+  axelarcli tx bitcoin verifyTx {verification info (output of previous cmd), e.g. "{\"OutPoint\":{\"Hash\":\"NxF/hGLGQZ6mTNyMWkYHPJ21E+2PMb1DV/beV7R9Gpk=\",\"Index\":1},\"Amount\":\"100000000\",\"BlockHash\":\"tPqsKekDOp5lW6QUl+YwlaD/3cmbQJwuUqgiNkqloQM=\",\"Address\":\"bcrt1qrnc097fuepeyrchganj4jzl2yuf5c0fg800uenr5h0d58emztxasusnk7p\",\"Confirmations\":\"21\"}"} --from validator -y -b block
+  -> wait for verification to be confirmed (~10 blocks)
+  ```
+- Trigger signing of all pending transfers to Ethereum
+  ```
+  axelarcli tx ethereum sign-pending-transfers --from validator -y -b block
+  -> returns commandID of signed tx
+  ```
```

Comment:
Warn about the need to wait for the signature to be voted

Reply:
done